pyluna-pathology/luna/pathology/cli/generate_mask.py

```diff
@@ -52,7 +52,7 @@ def generate_mask(input_slide_image, input_slide_roi, output_dir, annotation_nam
 
     Args:
         input_slide_image (str): path to slide image (.svs)
```

Comment:
Do we only support (.svs)? should we say (openslide compatible file format) for all input_slide_image args